src/Oscar.jl

```diff
@@ -151,7 +151,7 @@ function build()
   system("Build.jl")
 end
 
-
+include("Assertions.jl")
```

Comment:
I was not sure where to put this. Let me know if there is a more appropriate place.

Reply:
place is fine, but the filename should be lowercase

Reply:
It was uppercase in Hecke. But I can change it.

Reply:
yeah Hecke, AA, Nemo (still) do many things a bit different, and also Oscar is not yet consistent, but we kinda agree that filenames should use `snake_case` and not `kebab-case` or `CamelCase`. But of course we have dozens and dozens of files with existing names; there is no strong reason to rename them all; and if inside a directory *all* files already use `CamelCase`, I am fine with new files also using that. But for the files directly inside `src`, we are using `snake_case`, so this file should as well (of course `src/Oscar.jl` is the sole exception, but we don't have a choice there).

Reply:
Even if the filename is the name of a type? I somehow missed that discussion.